.bazelrc

```diff
@@ -46,7 +46,6 @@ build:crosslinuxarm --config=cross
 build:devdarwinx86_64 --platforms=//build/toolchains:darwin_x86_64
 build:devdarwinx86_64 --config=dev
 build:dev --define cockroach_bazel_dev=y
-build:dev --stamp --workspace_status_command=./build/bazelutil/stamp.sh
```

Comment:
Two problems with this:

1. In my testing, `cockroach` binaries can't start properly if they don't have the various stamped variables. You should be able to trivially reproduce this, but let me know if you can't. You can make this PR mergeable by removing those checks from `cockroach` (maybe only for `dev` builds if you need to?)
2. `dev doctor` will fail if you delete this line without making any other changes.